Article/Article.md

```diff
@@ -22,110 +31,80 @@ abstract: |
-
-<!--Furthermore, the "time since women suffrage" index was introduced by the authors to track long-lasting effects of the right to vote on gender equality, yet without any supporting information. The assumption that suffrage played a long-lasting effect on the balance in gender equality sounds reasonable but requires further investigation to be used as a robust estimator. 
--->
+$avgGenderDiff_{country} = \alpha + \beta_1 \ logGDPpc_{country} + \beta_2 \ genderEquality_{country}$
 
+Alternatively, to separate the contribution of the gender differences regressed on economic development, one can perform a regression conditioning on the gender quality indicator and then conditioning on economic development (Log GDP p/c). The theorem from Frisch–Waugh–Lovell [@10.2307/1907330; @Lovell] guarantees that the coefficients found from this conditional analysis are the same as the ones found for a regression of the gender differences on both economic development and gender equality index of the countries.
 
-Based on the identified methodological flaws and absence of validation and justification, we doubt the possibility to use Gender Equality Index as a reliable proxy for gender equality. Therefore, we will use GEI only to match the replication with the original analysis. The new analysis will use only established WEF GGG Index and the UNDP GII, in their relationship to the gender differences in economic preferences and the economic development.
+Once can also study the correlation between the gender differences and economic development without conditioning on gender equality index. Similarly, the relation between the gender differences and gender equality can be tested. Nevertheless, if the correlation between economic develpment and gender equality index is present, one can not derive a conclusing about their separate contibution without performing conditional analysis. We regressed the Log GDP p/c and joint gender equality index used by authors in the origninal article, toghether with three others gender quality indexes WEF, UNDP and GDI. The correlation between economic development expressed in Log GDP p/c and gender equality indexes in the countries is very strong (see Fig. 2), (r = , P , r = , P , r = , P , r = , P, respectively). Thus, Log GDP p/c can be used as a rather good estimator of any gender equality index. Because of this fact, the presence of strong correlation between gender differences in preference(s) and gender equality indexes is expected only if two variables are considered in the regression at the same time.
```

Comment:
Rewrite this as:
To check the correlation between economic development and gender equality of the countries, we regressed the Log GDP p/c and joint gender equality index used by authors in the original article, together with three others gender quality indexes WEF, UNDP and GDI. The correlation between economic development expressed in Log GDP p/c and gender equality indexes in the countries is very strong (see Fig. 2), (r = , P , r = , P , r = , P , r = , P, respectively). Thus, Log GDP p/c can be used as a rather good estimator of any gender equality index. Because of this fact, the presence of a strong correlation between gender differences in preference(s) and gender equality indexes is expected only if two variables are considered in the regression at the same time.

Reply:
Add citation: https://reports.weforum.org/global-gender-gap-report-2015/the-case-for-gender-equality/
76 counties only which participated in study